cmd/internal/cli/build.go

```diff
@@ -21,32 +21,26 @@ import (
 	"github.com/sylabs/singularity/pkg/cmdline"
 )
 
-var (
-	remote         bool
-	buildArch      string
-	builderURL     string
-	detached       bool
-	libraryURL     string
-	isJSON         bool
-	sandbox        bool
-	force          bool
-	update         bool
-	noTest         bool
-	sections       []string
-	noHTTPS        bool
-	tmpDir         string
-	dockerUsername string
-	dockerPassword string
-	dockerLogin    bool
-	noCleanUp      bool
-	fakeroot       bool
-	encrypt        bool
-)
+var buildArgs struct {
```

Comment:
move all the package level variables into a single struct, just to make the code easier to read in term of where things are coming from